docs/starters.yml

```diff
@@ -2239,3 +2239,25 @@
     - Optimized images.
     - Offline capabilities.
     - Auto-generated sitemap and robots.txt.
+- url: https://gatsby-starter.haezl.at
+  repo: https://github.com/haezl/gatsby-starter-haezl
+  description: A lightweight, mobile first blog starter with infinite scroll and Material-UI design for Gatsby.
+  tags:
+    - Blog
+    - Landing Page
```

Comment:
We currently don't use tag like that, so keeping this just in features and removing from tags would be best.